homeassistant/components/sleepiq/light.py

```diff
@@ -0,0 +1,68 @@
+"""
+Support for switching lights on Sleep Number beds off and on.
+
+Creates entities for both night stand lamps and the two
+night light (underbed) lights. Night lights cannot be
+controlled separately by the stock remote but can be with
+this integration.
+"""
+
+import logging
+
+from homeassistant.components import sleepiq
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = "mdi:lamp"
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the SleepIQ lights."""
+    if discovery_info is None:
+        return
+
+    data = sleepiq.DATA
```

Comment:
It looks like this will already be up to date in __init__ so you shouldn't need to update again here.

I realize this is existing in __init__ but we shouldn't use globals. You can fix this by storing the data in 
```
hass.data[DOMAIN] = DATA
```

Then you can access it in setup_platform

```
DATA = hass.data[DOMAIN]
```

`DATA` should probably be renamed as well.